Update address-rewriting.md (#3130)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/architecture/edge-transport-servers/address-rewriting.md

> [!IMPORTANT]
> Regardless of how you plan to use address rewriting, you need to verify that the resulting email addresses are unique in your organization so you don't end up with duplicates. Address rewriting doesn't verify the uniqueness of a rewritten email address.

> [!IMPORTANT]

> > DKIM signing is not supported for outboud email messages using rewriten addresses as it happens before the address rewriting process. If your organization requires DKIM signing for outbound email, plesase take in to account that DKIM validation will fail after implementing address rewriting.

Reviewer: ```suggestion
> DKIM signing is not supported for outbound email messages using rewritten addresses as it happens before the address rewriting process. If your organization requires DKIM signing for outbound email, be aware that DKIM validation will fail after implementing address rewriting.
```
